app/models/user_dashboard.rb

```diff
@@ -0,0 +1,21 @@
+class UserDashboard
```

Comment:
Great use of a facade/presenter 👍 

You should store this in a separate directory called `presenters` or `facades` instead of `models`.